spec/features/admin_can_view_submission_errors_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.feature 'Admin users can' do
+  before do
+    supplier = FactoryBot.create(:supplier, name: 'First Supplier')
+    task = FactoryBot.create(:task, supplier: supplier)
+    FactoryBot.create(
+      :submission_with_invalid_entries,
+      supplier: supplier,
+      task: task
+    )
+    sign_in_as_admin
+  end
+
+  scenario 'view a paged list of suppliers' do
```

Comment:
I'd like this scenario to be more descriptive of what it's testing - 'view the errors for submissions that failed validation' maybe?